library/src/window_impl.cxx

```diff
@@ -35,6 +36,28 @@ class window_impl::internals
   {
   }
 
+  std::string GetCachePath()
+  {
+#if defined(_WIN32)
+    std::string cachePath = vtksys::SystemTools::GetEnv("LOCALAPPDATA");
+    cachePath = cachePath + "/f3d";
+#else
+    std::string cachePath = vtksys::SystemTools::GetEnv("HOME");
+#if defined(__APPLE__)
+    cachePath = cachePath + "/Library/Caches/f3d";
+#elif defined(__unix__)
+    cachePath = cachePath + "/.cache/f3d";
```

Comment:
I wonder if this should be documented somewhere ?

Reply:
either in F3D or libf3d doc

Reply:
BTW the location should be set by the application I think

Reply:
We need a default location, so it has to be defined in libf3d. I'll add a public API to configure it though.